reference/6/Microsoft.PowerShell.Management/Test-Connection.md

```diff
@@ -15,151 +15,162 @@ Sends ICMP echo request packets ("pings") to one or more computers.
+Test-Connection [-TargetName] <string[]> -Traceroute [-IPv4] [-IPv6] [-ResolveDestination]
+  [-Source <string>] [-MaxHops <int>] [-TimeoutSeconds <int>] [-Quiet] [<CommonParameters>]
 ```
 
-### Quiet
+### ConnectionByTCPPort
+
 ```
-Test-Connection [-DcomAuthentication <AuthenticationLevel>] [-WsmanAuthentication <String>]
- [-Protocol <String>] [-BufferSize <Int32>] [-ComputerName] <String[]> [-Count <Int32>]
- [-Impersonation <ImpersonationLevel>] [-TimeToLive <Int32>] [-Delay <Int32>] [-Quiet]
- [<CommonParameters>]
+Test-Connection [-TargetName] <string[]> -TCPPort <int> [-IPv4] [-IPv6] [-ResolveDestination]
+  [-Source <string>] [-TimeoutSeconds <int>] [-Quiet] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-Connection** cmdlet sends Internet Control Message Protocol (ICMP) echo request packets, or pings, to one or more remote computers and returns the echo response replies.
-You can use this cmdlet to determine whether a particular computer can be contacted across an IP network.
 
-You can use the parameters of **Test-Connection** to specify both the sending and receiving computers, to run the command as a background job, to set a time-out and number of pings, and to configure the connection and authentication.
+The **Test-Connection** cmdlet sends Internet Control Message Protocol (ICMP) echo request packets,
+or pings, to one or more remote computers and returns the echo response replies. You can use this
+cmdlet to determine whether a particular computer can be contacted across an IP network.
+
+You can use the parameters of **Test-Connection** to specify both the sending and receiving
+computers, to run the command as a background job, to set a time-out and number of pings, and to
+configure the connection and authentication.
 
-Unlike the familiar **ping** command, **Test-Connection** returns a **Win32_PingStatus** object that you can investigate in PowerShell.
-You can use the *Quiet* parameter to force it to return only a **Boolean** value.
+Unlike the familiar **ping** command, **Test-Connection** returns a
+**TestConnectionCommand+PingReport**. You can use the **Quiet** parameter to force it to return
+only a **Boolean** value.
 
 ## EXAMPLES
 
 ### Example 1: Send echo requests to a remote computer
+
+This command sends echo request packets from the local computer to the Server01 computer.
+
+```powershell
+Test-Connection Server01 -IPv4
 ```
-PS C:\> Test-Connection "Server01"
 
-Source        Destination     IPV4Address     IPV6Address  Bytes    Time(ms)
-------        -----------     -----------     -----------  -----    --------
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       0
-ADMIN1        Server01         10.59.137.44                32       1
+```Output
+Pinging Server01 [172.16.0.55] with 32 bytes of data:
```

Comment:
All other examples for Server01 use IP 10.59.137.44. Should we keep it consistent for any doing comparisons?

Reply:
Fixed